cmd/wallet-web/src/pages/OIDCShare.vue

```diff
@@ -269,7 +270,11 @@ export default {
         console.error('get credentials failed,', e);
         this.loading = false;
       }
-      this.generateIdToken();
+      if (this.verifyRPRegistration()) {
+        this.generateIdToken();
+      } else {
+        this.errors.push('RP not registered and Id token cannot be generated');
```

Comment:
Looks like we are missing a verb, maybe something like the following?
```suggestion
        this.errors.push('RP is not registered and Id token cannot be generated');
```